analysis/graphs/percent_plt_reduction/compute_median_reduction.rb

```diff
@@ -14,7 +14,7 @@ def is_pc(experiment_name)
   # TODO(cs): encoding the experiment in the filename is jenky.
   # Should really use a database.
   experiment_name = experiment_name.gsub(/.\d+.har$/, "")
-  return (experiment_name =~ /.pc$/)
+  return (experiment_name.include?(".pc"))
```

Comment:
@colin-scott I realize that your line works if the filename *ends* in `.pc`, but this isn't the case for my files, as they end with `.pc.har`, this takes care of both cases